src/constructs/lambda/lambda.ts

```diff
@@ -141,6 +142,7 @@ export class GuLambdaFunction extends Function {
     const code = Code.fromBucket(bucket, objectKey);
     super(scope, id, {
       ...props,
+      logFormat: !logFormat ? "JSON" : logFormat,
```

Comment:
[Destructuring with assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) would make this more readable (vs a negated ternary). It also looks like AWS CDK have an [enum](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.LoggingFormat.html) available:

```ts
import { LoggingFormat } from "aws-cdk-lib/aws-lambda";

const {
  logFormat = LoggingFormat.JSON
} = props;
```

Reply:
These are non-blocking comments.